File: Data/DataModels/CryptoTradeHistory.cs
using System;

namespace Data.DataModels
{
    public class CryptoTradeHistory
    {
        public int Id { get; set; }

        public DateTime TradeTimeStamp { get; set; }

        public int CryptoTickerId { get; set; }

        public CryptoTicker CryptoTicker { get; set; }

        public double TradeValue { get; set; }

        public int ValueCurrencyId { get; set; }

        public CurrencySymbol CurrencySymbol { get; set; }
    }
}

File: BudgetManager.WebScrapers/Orm/CryptoTradeHistory.py
from sqlalchemy import Column, Integer, Float, DateTime, Sequence
from sqlalchemy.ext.declarative import declarative_base

Base = declarative_base()

class CryptoTradeHistory(Base):
    __tablename__ = 'CryptoTradeHistory'
    Id = Column(Integer, Sequence('id_seq'), primary_key=True)
    TradeTimeStamp = Column(DateTime, nullable=False)
    CryptoTickerId = Column(Integer, nullable=False)
    TradeValue = Column(Float, nullable=False)
    CurrencySymbolId = Column(Integer, nullable=False)
    TradeSize = Column(Float, nullable=False)
    UserIdentityId = Column(Integer, nullable=False)